cdk/lib/amiable/__snapshots__/amiable.test.ts.snap

```diff
@@ -882,3 +882,1085 @@ Object {
   },
 }
 `;
+
+exports[`The Amiable stack matches the snapshot 2`] = `
```

Comment:
This is brand new because we weren't testing the `PROD` snapshot at all before. For the changes related to this PR, see: https://github.com/guardian/amiable/commit/6c4055331d1309d01a89b53a6c0a3035c8b47e82.